src/bda/plone/shop/user/userdata.py

```diff
@@ -166,11 +166,13 @@ class UserDataSchemaAdapter(AccountPanelSchemaAdapter):
 
     @property
     def fullname(self):
+        # TODO: I think this never gets called as there
+        # was an error in the string formatting.
```

Comment:
@thet what was this code good for?

https://github.com/bluedynamics/bda.plone.shop/blame/master/src/bda/plone/shop/user/userdata.py#L167

You implemented exactly the computed properties ``location`` and ``fullname`` in UserDataSchemaAdapter which are now in the custom PAS plugin.

Did the lookup in the related plone views change?